content/html/concepts/elements/elements.md

```diff
@@ -59,8 +65,10 @@ HTML is organized into a family tree structure. HTML elements can have parents,
 </body>
 ```
 
-So here are the relationships for `<h1>`:
+Here are the relationships depicted above in relation to the <h1> tag:
```

Comment:
```suggestion
Here are the relationships depicted above in relation to the '<h1>' tag:
```
Add '' around h1 header otherwise all text after will be written in header syntax.